Bots/DotNet/Skills/Composer/EchoComposerSkillBot/runtime/customaction/Action/MultiplyDialog.cs

```diff
@@ -0,0 +1,68 @@
+using System;
```

Comment:
The copyright header is missing here.

Reply:
Opened an issue with composer reporting this (https://github.com/microsoft/BotFramework-Composer/issues/4923)